### ISSUE TITLE
Google Analytics Setup

### ISSUE THREAD

**Issue by skanda-vasishta** [ioos/sanctuarywatch#74]
### What should we add?

Gtag.js or GTM implementation needs to be added.

### Reference

- https://developers.google.com/tag-platform/tag-manager/datalayer
- https://support.google.com/tagmanager/answer/7679320?hl=en&ref_topic=7679108
- https://support.google.com/tagmanager/answer/7679219?hl=en
- https://support.google.com/tagmanager/answer/7182738
- https://www.simoahava.com/analytics/simple-custom-event-listeners-gtm/

**Comment by robbiecarroll** [comment]
Question to get started:

- Do we have a google analytics/ tag manager account already?
- Can I have access to it?
- I believe this will help me with testing a debugging.

**Comment by superjai** [comment]
Good point. I just made you an admin on the Sanctuary Watch analytics/tag manager accounts. Hopefully you got the emails from google with the invites. I should have done that earlier.
